FogCreek/Glitch-Community
# Persistent papaya: Fix some sentry errors

## Links
https://persistent-papaya.glitch.me/
https://app.clubhouse.io/glitch/story/4081/typeerror-cannot-read-property-includes-of-undefined
https://app.clubhouse.io/glitch/story/4517/signing-in-without-a-destination-set-throws-an-error
https://app.clubhouse.io/glitch/story/4083/missing-null-check-in-redirect-to-destination

## Changes:
- Updated the sentry api to the latest version
- Added null checks to sentry and sign in destinations
- Data loaders always use get/args to prevent re-renders

## How To Test:
- Make sure the site still remembers where you were after signing in
- Load up a /user/:id page, a project page, and a collection page
